subxt/src/backend/unstable/mod.rs

```diff
@@ -407,17 +406,15 @@ impl<T: Config + Send + Sync + 'static> Backend<T> for UnstableBackend<T> {
                     Some(ev) => ev,
                 };
 
-                let runtime_details = match runtime_event {
+                      let runtime_details = match runtime_event {
```

Comment:
the indentation looks strange, is mixing spaces/tabs perhaps?

Reply:
Oh, this is rustfmt giving up formatting [(without warning! see this discussion)](https://users.rust-lang.org/t/rustfmt-silently-fails-to-work/75485/4), if a single expression cannot be formatted to fit in the line length limit.